hypervisor/include/arch/x86/asm/cpu.h

```diff
@@ -487,11 +487,11 @@ void wait_sync_change(volatile const uint64_t *sync, uint64_t wake_sync);
 	asm volatile ("movdqu %%" STRINGIFY(xmm) ", %0": "=m" (*(result_ptr)));	\
 }
 
-#define CPU_XMM_WRITE(xmm, value)					\
+#define CPU_XMM_WRITE(xmm, input_ptr)					\
```

Comment:
rename input_ptr to input___m128i_p ?

Reply:
And rename result_ptr in CPU_XMM_READ to result_m128i_p ?

Reply:
updated.